packages/groovy/build.sh

```diff
@@ -2,7 +2,7 @@ TERMUX_PKG_HOMEPAGE=https://groovy-lang.org/
 TERMUX_PKG_DESCRIPTION="A powerful multi-faceted programming language for the JVM platform"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=4.0.0
+TERMUX_PKG_VERSION=4.0.1
 TERMUX_PKG_REVISION=1
```

Comment:
Remove this line (`TERMUX_PKG_REVISION`) when bumping version.

Reply:
Oh, I didn't notice that! :(